contrib/gomodule/redigo/redigo.go

```diff
@@ -158,5 +168,50 @@ func (tc Conn) Do(commandName string, args ...interface{}) (reply interface{}, e
 		}
 	}
 	span.SetTag("redis.raw_command", b.String())
-	return tc.Conn.Do(commandName, args...)
+	return do(commandName, args...)
+}
+
+// Do wraps redis.Conn.Do. It sends a command to the Redis server and returns the received reply.
+// In the process it emits a span containing key information about the command sent.
+// When passed a context.Context as the final argument, Do will ensure that any span created
+// inherits from this context. The rest of the arguments are passed through to the Redis server unchanged.
+func (tc Conn) Do(commandName string, args ...interface{}) (reply interface{}, err error) {
+	return withSpan(
```

Comment:
You can just do:
```
return withSpan(tc.Conn.Do, tc.params, commandName, args...)
```
The same for the other one:
```
return withSpan(tc.ConnWithTimeout.Do, tc.params, commandName, args...)
```

Reply:
very good point, this can be simplified